tests/dynamic_data/users_roles/delete_user_test.py

```diff
@@ -131,14 +131,17 @@ def test_delete_own_public_publications(publication_type):
 @pytest.mark.parametrize('publication_type', process_client.PUBLICATION_TYPES)
 @pytest.mark.usefixtures('ensure_layman_module', 'oauth2_provider_mock')
 def test_delete_shared_publications(publication_type):
-    # create publication with different users write access rights
+    # create publication with different users, roles write access rights
```

Comment:
Rename `test_delete_shared_publications_with_readers` to `test_delete_user_with_undeletable_publications`